### DISCUSSION TITLE
Doubt setting path for clip_l & t5xxl.

### DISCUSSION THREAD

**Discussion by ZeroCool22** [mcmonkeyprojects/SwarmUI#224] [Q&A]
I already have FORGE and AUTO's1111 so i want to use the 2 files i already have on that folders.

It's the correct path to use?:

![ssssddddd](https://github.com/user-attachments/assets/988973c0-c021-43c1-bc7c-38f4957569f1)

Or i need to copy **clip_**l & **t5xxl** to **models/clip of Swarm**?

Thx.

**Comment by mcmonkey4eva** [comment]
post on the issues page to request support for changing the clip folder path
